core/templates/components/question-difficulty-selector/question-difficulty-selector.directive.html

```diff
@@ -0,0 +1,40 @@
+<md-radio-group ng-model="skillWithDifficulty._difficulty">
+  <md-radio-button value="0.3">
```

Comment:
There's a lot of repetition here -- consider structuring using an ng-repeat instead?

Reply:
Done